src/ext/java/org/opentripplanner/ext/vehicleparking/parkapi/ParkAPIUpdater.java

```diff
@@ -85,8 +101,8 @@ protected VehicleParking parseElement(JsonNode jsonNode) {
       .state(state)
       .x(x)
       .y(y)
+      .openingHoursCalendar(parseOpeningHours(jsonNode.path("opening_hours"), vehicleParkId))
       // TODO
-      // .openingHours(parseOpeningHours(jsonNode.path("opening_hours")))
       // .feeHours(parseOpeningHours(jsonNode.path("fee_hours")))
```

Comment:
Forgot to mention about this. I'm not sure what kind of a structure we should have for the fee hours as there can be different types of fares that are required at different times. HSL park data includes that information (as of now, I don't need to parse it to OTP though). Although, the same problem is with opening hours as well as something can be only open for students or open at limited capacity etc. OSM includes comments for opening hour rules but it's hard to structure the opening hours around the comments in UIs and also they lack translations.